app/build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
 }
 
 def vCode = "${System.env.VERSION_CODE ?: 1}".toInteger()
-def vName = "1.8.0-${vCode}"
+def vName = "1.9.0-${vCode}"
```

Comment:
When we go to release to production we bump the version number, no need to do it in this PR.

Reply:
No Problem, I removed the last commit.